src/picongpu/include/simulation_defines/param/radiationConfig.param

```diff
@@ -105,32 +108,59 @@ namespace picongpu
 
     namespace parameters
     {
-        /*!enable radiation calculation*/
+/*! enable radiation calculation
+ *
+ * If radiation is enabled the attribute `momentumPrev1` needs to be added to electron species
+ */
 #define ENABLE_RADIATION 0
 
-        // Nyquist low pass allows only amplitudes for frequencies below Nyquist frequency
-        // 1 = on  (slower and more memory, no fourier reflections)
-        // 0 = off (faster but with fourier reflections)
+// Nyquist low pass allows only amplitudes for frequencies below Nyquist frequency
+// 1 = on  (slower and more memory, no fourier reflections)
+// 0 = off (faster but with fourier reflections)
 #define __NYQUISTCHECK__ 0
 
-
-
-        //marked every N particle for calculate radiation
-        //mark no particle at beginning
-#define RAD_MARK_PARTICLE 1
-
-        //disable or enable gamme filter
-        //diable (0) / enable (1)
-#define RAD_ACTIVATE_GAMMA_FILTER 0
-
-        constexpr float_32 RadiationGamma = 5.;
-
         constexpr unsigned int N_observer = 256; // number of looking directions
 
 
 
     } /* end namespace parameters */
 
+    namespace radiation
+    {
+        /** activate particles for radiation */
+        struct GammaFilterFunctor
+        {
+            static constexpr float_X radiationGamma = 5.0;
+
+            template< typename T_Particle >
+            DINLINE void operator()( T_Particle& particle )
+            {
+                if(
+                    picongpu::gamma<float_X>(
+                        particle[ picongpu::momentum_ ],
+                        picongpu::traits::attribute::getMass(
+                            particle[ picongpu::weighting_ ],
+                            particle
+                        )
+                    ) >= radiationGamma
+                )
+                particle[ picongpu::radiationFlag_ ] = true;
```

Comment:
QUESTION:
How do you make sure it is initialized as `false`?
(I could not find the corresponding param file.)